src/site/content/en/blog/persistent-storage/index.md

```diff
@@ -0,0 +1,166 @@
+clear storage. Thus if a user visits your site regularly, the chances are
+small that your data will be evicted.
+
+To prevent the browser from deleting your data, you can request that your
+entire site's storage be marked persistent. **You should only do this if you
+have critical data that is not backed up to the cloud, for example end-to-end
+encryption keys, that if deleted could result in significant data loss.**
+Persistent storage will not be deleted by the browser, even if storage on the
+device is running tight. It will only be deleted if the user chooses to
+remove it via their site settings.
+
+Persistent storage is [supported in many modern][caniuse-persistent] browsers.
+
+{% Aside %}
+  To learn more about eviction, how much you can store, and how to handle quota
+  limitations, see [Storage for the web](/storage-for-the-web/).
+{% endAside %}
+
+## Check if your sites storage has been marked as persistent
+
+You can use JavaScript to determine if your site's storage has been marked
+as persistent. Calling `navigator.storage.persisted()` returns a Promise that
+resolves with a boolean, indicating whether storage has been marked as
+persisted.
+
+```js
+// Check if site's storage has been marked as persistent
+if (navigator.storage && navigator.storage.persist) {
+  const isPersisted = await navigator.storage.persisted();
+  console.log(`Persisted storage granted: ${isPersisted}`);
+}
+```
+
+## When should I ask for persistent storage?
+
+The best time to request your storage be marked as persistent is when you
+save critical user data, and the request should ideally be wrapped in a user
+gesture. Do not ask for persistent storage on page load, or in other bootstrap
+code, the browser may prompt the user for permission. If the user
+isn't doing anything that they think needs to be saved, the prompt may be
+confusing, and they'll likely reject the request. Additionally, don't prompt
+too frequently, if the user decided not to grant permission, don't immediately
+prompt again on the next save.
+
+## Request persistent storage
+
+To request persistent storage for your site's data, call
+`navigator.storage.persist()`. It returns a Promise that resolves with a
+boolean, indicating whether the persistent storage permission was granted.
+
+```js
+// Request persistent storage for site
+if (navigator.storage && navigator.storage.persist) {
+  const isPersisted = await navigator.storage.persist();
+  console.log(`Persisted storage granted: ${isPersisted}`);
+}
+```
+
+{% Aside %}
+  The API names to *check* if your site's storage has already been marked
+  persistent, and to *request* persistent storage are very similar. The way
+  I remember the difference is `persisted()` is past-tense, and is used to
+  check if it's already persist**ed**. Whereas `persist()` is present-tense
+  and asks for it now.
+{% endAside %}
+
+### How is permission granted?
+
+Persistent storage is technically granted as a permission. Browsers use
```

Comment:
Can you append with something like:

>Persistent storage is technically granted as a permission **, which is...[tech tech tech]**.

This and the title above read too much like a tautology.

Reply:
Rewrote this bit.